resources/standards-data/language-subtag-registry/language-subtag-registry

```diff
@@ -1,4 +1,4 @@
-File-Date: 2024-05-16
+File-Date: 2025-02-06
```

Comment:
Can you also update the minimum versions file and regenerate?

Reply:
Good catch!